band_together/entities/enemies/crab/crab.gd

```diff
@@ -2,38 +2,102 @@ extends CharacterBody2D
 
 # I FOLLOWED THIS TUTORIAL
 # https://www.youtube.com/watch?v=P02PcfgqrY8&ab_channel=GameDevKnight
```

Comment:
What if we put this in the PR description instead? Or we could make a google doc with all the helpful tutorials we followed.